### ISSUE TITLE
[RTL] Text from "Secondary email" option in Firefox accounts section is not translated [bz1463762]

### ISSUE THREAD

**Issue by bug-mirror** [mozilla/fxa-bugzilla-mirror#505]
From https://bugzilla.mozilla.org/show_bug.cgi?id=1463762

Created attachment 8979961
RTL secondary email.png

Device(s):
 -  Sony Xperia Z5(Android 7.0);
 -  One Plus 3 (Android 6.0.1);

Build(s):
 - Nightly 62.0a1 (2018-05-23);
 - Beta 61.0b7
 - Release 60.0.1

Steps to reproduce:
 1. Change language to Arabic.
 2. Access the Firefox account menu.
 3. Tap on the Add button from the Secondary email option

Expected result:
Text is correctly translated

Actual result:
Text still appears in English

**Comment by shane-tomlinson** [comment]
These strings are not translated to [Arabic in Pontoon](https://pontoon.mozilla.org/ar/firefox-accounts/LC_MESSAGES/client.po/?string=134220). I double checked [the template](https://github.com/mozilla/fxa-content-server/blob/821f27649fff0ab0d5d3c736b364588fcbf1d3b7/app/scripts/templates/settings/emails.mustache#L15) to ensure this strings is wrapped as expected, which it is. Rather than any action that needs done by the FxA dev team, we need the strings translated.

**Comment by davismtl** [comment]
Before translating, we should maybe address this to avoid duplicate effort:
https://bugzilla.mozilla.org/show_bug.cgi?id=1464762 (which has strangely a near identical bug number)

**Comment by shane-tomlinson** [comment]
> Before translating, we should maybe address this to avoid duplicate effort:
https://bugzilla.mozilla.org/show_bug.cgi?id=1464762 (which has strangely a near identical bug number)

@davismtl - These seem to me two separate flows. This bug refers to untranslated text in the setting panel, while the bug you referenced refers to confusion in the screen that is displayed after verifying a signin.

**Comment by davismtl** [comment]
you're right @shane-tomlinson we noticed that in triage this week. Please ignore that comment.
